dashboard/app/models/ability.rb

```diff
@@ -121,6 +121,9 @@ def initialize(user)
         ReviewableProject.user_can_mark_project_reviewable?(project_owner, user)
       end
       can :destroy, ReviewableProject, user_id: user.id
+      can :project_commits, ProjectVersion do |_, project_owner, project_id|
+        CodeReviewComment.user_can_review_project?(project_owner, user, project_id)
```

Comment:
We'll be deleting the `code_review_comments` table and associated model because it stores the comments for code review v1. @jamescodeorg created the new tables, we'll probably want to add a similar method (`user_can_review_project`) to the CodeReviewRequests model (I don't think this exists yet) and use that instead

Reply:
Do you want me to copy it over in this PR? I like this shared method because it abstracts away a lot of this logic -- apart from a couple tests, I don't have to worry about code review groups -- but it's a good point that it's going away soon.

Reply:
No need to do it in this PR, but we'll need to do it eventually for this endpoint to work as expected (also because ReviewableProject is going away for v2 so the logic in `user_can_review_project` will need to be updated). If you're not doing it now, maybe make a ticket for it so we don't forget to update

Reply:
https://codedotorg.atlassian.net/browse/LP-2357. It might be something that could be moved to a helper file as well.